<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckExportWizard</class>
 <widget class="QWizard" name="DeckExportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Deck</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../deckeditor.qrc">
    <normaloff>:/DeckEditor/Resources/Icons/box.png</normaloff>:/DeckEditor/Resources/Icons/box.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnLastPage|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <property name="titleFormat">
   <enum>Qt::PlainText</enum>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::PlainText</enum>
  </property>
  <widget class="QWizardPage" name="wizardSettings">
   <property name="title">
    <string>Export Project as .deck File</string>
   </property>
   <property name="subTitle">
    <string>Please provide the required information to proceed with exporting your Deck.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>API Version 1.0</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Export Deck with API 1.0 (deprecated)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Compatible to Arcomage Tribute Versions 0.2.0.0  through 0.4.9.0.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Although deprecated, this is the currently preferred format.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>API Version 1.1</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export Deck with API 1.1 (disabled)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>The new standard format, currently disabled until Arcomage Tribute 0.5.0.0+ is in Beta State and publicly available.</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select an API Version:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="3">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="buttonSelectFile">
      <property name="text">
       <string>Select</string>
      </property>
      <property name="icon">
       <iconset resource="../deckeditor.qrc">
        <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Filename:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="file">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="noPrompt">
      <property name="text">
       <string>Overwrite existing Files without prompt</string>
      </property>
      <property name="icon">
       <iconset resource="../deckeditor.qrc">
        <normaloff>:/DeckEditor/Resources/Icons/exclamation-small.png</normaloff>:/DeckEditor/Resources/Icons/exclamation-small.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardExport">
   <property name="title">
    <string>Exporting...</string>
   </property>
   <property name="subTitle">
    <string>Please wait while the Wizard is exporting your Deck. You can view the Output below.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPlainTextEdit" name="exporterOutput">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../deckeditor.qrc"/>
 </resources>
 <connections/>
</ui>
